addons/web/static/src/views/calendar/calendar_common/calendar_common_popover.js

```diff
@@ -24,6 +24,10 @@ export class CalendarCommonPopover extends Component {
     get isEventDeletable() {
         return this.props.model.canDelete;
     }
+    get datatooltip() {
```

Comment:
```suggestion
    get dataTooltip() {
```

Reply:
Hey @xavierbol  the fields we utilized in the module's calendar view will have them records displayed in the calendar popover body. You can check it out under the [Record tag ](https://github.com/odoo/odoo/pull/138336/files#diff-62c83a09211a9f6a23a0ca089ec994344438dca5e60e1e2895ffd383b0a9c031R47). This is the reason we are getting field record in datatooltip()

Reply:
If that property is used outside the Record tag then fieldName and the getter will not work.
We should either set the variable inside the template or define a getter valid for all use cases and not only in a certain scope.